GOVERNANCE.md

```diff
@@ -19,7 +18,8 @@ which is guided by the following principles:
 * Comprehensive and accessible documentation and run-time signals (errors, warnings)
 * Clear error and warning messages
 
-To prioritize developer time, we define what is in and out of current scope.
+To prioritize developer time, we define what is in and out of current scope. Feature requests in issues and pull requests that are out of current scope should be closed immediately, because they are not the current priority. If someone wants to contribute code that is currently out of scope, they first have to make a pull request that changes the scope as defined below.
```

Comment:
here is the addition I made to clarify current scope and how to change it

Reply:
I like this. The process makes sense that we'd get to review this change before they put in a bunch of work on something that isn't wanted.